CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+# v23.0.0
+
+- [Components] Added new `ResetStyles` and `BrandStyles` components to inject global styles. These replace the `utils/injectGlobalStyles/style` exports that were used for the same purpose. ([#1051)](https://github.com/curology/radiance-ui/pull/1051))
```

Comment:
[Nit] This is technically a breaking change 😉

Reply:
It is but that file was never exported in a documented entrypoint so it's covered by the other breaking change line below 😁